tests/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ foreach(test_name aediis-1
                   fd-freq-energy fd-freq-energy-large fd-freq-gradient
                   fd-freq-gradient-large fd-gradient freq-isotope1 freq-isotope2 fnocc1 fnocc2
                   fnocc3 fnocc4 fnocc5 fnocc6 fnocc7 frac frac-ip-fitting frac-sym frac-traverse ghosts gibbs
-                  lccd-grad1 lccd-grad2 matrix1 matrix2
+                  large-atoms lccd-grad1 lccd-grad2 matrix1 matrix2
```

Comment:
I think I agree with Jonathon that as a fresh test, this is easier to read as a pytest. Particularly since now libecpint is an optional add-on (ecp capability was built-in when large-atoms was first drafted), this test would need to be moved to the `ENABLE_ecpint` section of this file for the extra logic for cmake to decide when it's runable. shifting to pytest is probably preferred.

Reply:
Alright, great. I'll switch this over to pytest then.